db/migrate/20250731150352_drop_initial_login_id_from_logins.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class DropInitialLoginIdFromLogins < ActiveRecord::Migration[7.2]
+  def change
+    # The column is ignored and all references have been removed
```

Comment:
Is this ignore already in production?

Reply:
It would be (see the big warning in the description). This would ship as the 4th PR.

Reply:
Perfect - thank you